ipaserver/install/server/replicainstall.py

```diff
@@ -1250,20 +1250,13 @@ def promote_check(installer):
                 'KRA', conn, config.kra_host_name)
         if kra_host is not None:
             config.kra_host_name = kra_host
-            kra_enabled = True
-        else:
-            if options.setup_kra:
-                root_logger.error("There is no KRA server in the domain, "
-                                  "can't setup a KRA clone")
-                raise ScriptError(rval=3)
-            kra_enabled = False
 
         if ca_enabled:
             options.realm_name = config.realm_name
             options.host_name = config.host_name
             ca.install_check(False, config, options)
 
-        if kra_enabled:
```

Comment:
Yes, to deepen my point - both should be handled the same way so I don't care one is being changed but the other has to be, too.